docs/helm/desktop-deployment.md

```diff
@@ -1,57 +1,53 @@
+### Rancher Desktop specific configuration
 
-![Resources](./diagrams/dfd-resources.png)
+Uncheck `Enable Traefik` from the `Kubernetes Settings` page to disable Traefik. You may need to exit and restart Rancher Desktop for the change to take effect. Ref: [Setup NGINX Ingress Controller](https://docs.rancherdesktop.io/how-to-guides/setup-NGINX-Ingress-Controller)
 
-### Docker for Desktop specific configuration
+### Docker Desktop specific configuration
 
-To deploy the Helm charts Kubernetes needs to be enabled, this can be done from the "Kubernetes" tab, as shown in the screenshot below. Press the "Apply & Restart" button to confirm.
+[Prerequisites](./desktop-deployment.md#Prerequisites) are consistent with those for Docker Desktop. Additionally, it is essential to install the latest version of [Kubectl](https://kubernetes.io/docs/tasks/tools/install-kubectl) & [Helm](https://helm.sh/docs/intro/install).
 
-![k8s Enabled](./diagrams/dfd-k8s-enabled.png)
+After the installation of Docker Desktop, the following configurations should be adjusted within Docker Desktop settings.
+`Settings > Resources > Advanced > CPUs:8, Memory: 16GB, Swap: 1GB`
+`Settings > kubernetes > Enable Kubernetes`
 
-### Rancher Desktop specific configuration
+After making the necessary settings `Apply and restart` the docker desktop.
 
-Uncheck `Enable Traefik` from the `Kubernetes Settings` page to disable Traefik. You may need to exit and restart Rancher Desktop for the change to take effect. Ref: [Setup NGINX Ingress Controller](https://docs.rancherdesktop.io/how-to-guides/setup-NGINX-Ingress-Controller)
+## Deployment
 
-## Deploy
+After successfully launching Rancher Desktop, proceed with the instructions provided in the following sections to install ACS (Enterprise or Community) on your local system.
 
-Once Docker for Desktop is running follow the steps in the following sections to deploy ACS (Enterprise or Community) to your local machine.
+> Note: To execute the deployment on Docker Desktop, please adhere to the same set of deployment instructions provided below.
 
 ### Namespace
 
-Namespaces in Kubernetes isolate workloads from each other, create a namespace to host ACS inside the cluster using the following command (we'll then use the `alfresco` namepsace throughout the rest of the tutorial):
+To establish an isolated environment for ACS within the Kubernetes cluster, initiate the creation of a Kubernetes namespace using the provided command. Throughout the subsequent sections of this tutorial, we will consistently refer to this namespace as 'alfresco'
 
 ```bash
 kubectl create namespace alfresco
 ```
 
 ### Ingress
 
-Add the chart repository using the following command:
+Utilize the subsequent command to include the chart repository:
```

Comment:
I don't like unnecessary wording before a command, I will prefer for readability:

```suggestion
Add the ingress-nginx chart repository:
```

it's already clear for me that it's a command because we are using code fences